.github/workflows/check_gpg_key_not_expired.yml

```diff
@@ -32,11 +67,39 @@ jobs:
         shell: bash
         run: |
           bash test/scripts/cert-harmony.sh
-          
-      - name: Discord notification
-        uses: Ilshidur/action-discord@0.3.2
-        if: failure()
-        env:
-          DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK }}
+
+      - uses: slackapi/slack-github-action@v1.27.0
+        if: always()
```

Comment:
Is this was you want?

IIRC, `always()` was counterintuitive to me. In some places, I had to use `failure() || success()`.

Reply:
yes, I want the action to update slack if the job is cancelled too